playbook/spec/pb_kits/playbook/kits/phone_number_input_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_popover/popover"
```

Comment:
Everything looks great! Small thing on this spec test: shouldn't this require_relative line require the phone_number_input kit, not popover? cc: @jasperfurniss